digitalpublicgoods/vaccineledger.json

```diff
@@ -0,0 +1,102 @@
+{
+  "name": "VaccineLedger",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://gitlab.com/statwig-public/theledger/-/blob/development/LICENSE"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://gitlab.com/statwig-public/theledger",
+      "https://gitlab.com/statwig-public/theledger/-/blob/development/docs/VaccineLedger-_Use_Case___Functional_Features_.pdf",
```

Comment:
A PDF does not feel like an accessible or effective method to deliver open source documentation, but I guess it meets the mark.

Reply:
@jwflory this is a good suggestion for VaccineLedger to consider when adopting more accessible formats for their open source docs, but for now their documentation passes the DPG standard.